tests/acceptance/authentication-qunit-test.js

```diff
@@ -0,0 +1,91 @@
+import { tryInvoke } from '@ember/utils';
+import {
+  currentURL,
+  setupContext,
+  setupApplicationContext,
+  teardownApplicationContext,
+  teardownContext,
+  visit
+} from '@ember/test-helpers';
+import hasEmberVersion from 'ember-test-helpers/has-ember-version';
+import {
+  describe,
+  it,
+  beforeEach,
+  afterEach
+} from 'mocha';
+import { expect } from 'chai';
+import Pretender from 'pretender';
+import {
+  invalidateSession,
+  authenticateSession,
+  currentSession
+} from 'ember-simple-auth/test-support';
+import config from '../../config/environment';
+
+describe('Acceptance: Authentication (for QUnit helpers)', function() {
```

Comment:
Just a nit-pick, but those helpers are not QUnit specific (and as a matter of fact, in there they are working in mocha).
I'd name the module something different, like "New-style Test Helpers"

Reply:
@cibernox I'll rename the file and `describe('Acceptance: New-style Test Helpers)'.